### PR TITLE
Derive more ToCss impls

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/17382)

### CODE REVIEW

**Comment by nox** [comment]
r? @emilio 
@bors-servo retry
https://treeherder.mozilla.org/#/jobs?repo=try&revision=fd326b15625d143d9adc237bd6ca9e4d345d2dcf

File: components/style/font_face.rs [hunk 1]
```diff
@@ -24,33 +24,17 @@ use style_traits::{ToCss, OneOrMoreCommaSeparated, ParseError, StyleParseError};
 use values::specified::url::SpecifiedUrl;
 
 /// A source for a font-face rule.
-#[derive(Clone, Debug, PartialEq, Eq)]
 #[cfg_attr(feature = "servo", derive(Deserialize, Serialize))]
+#[derive(Clone, Debug, Eq, PartialEq, ToCss)]
 pub enum Source {
     /// A `url()` source.
+    #[css(function)]
```
Comment (emilio) [inline components/style/font_face.rs]: This won't add the quotes, right?

**Review by emilio — APPROVED** [review]
r=me modulo that bit, which isn't idempotent.

File: components/style/servo/url.rs [hunk 2]
```diff
@@ -123,7 +121,8 @@ impl ToCss for SpecifiedUrl {
             }
         };
 
-        try!(CssStringWriter::new(dest).write_str(string));
+        dest.write_str("url(")?;
+        string.to_css(dest)?;
         dest.write_str("\")")
```
Comment (emilio) [inline components/style/servo/url.rs]: no need for the trailing quote now :)

**Comment by nox** [comment]
@bors-servo r=emilio

Unused imports.

**Pull request merged**